R/Spectra.R

```diff
@@ -389,8 +389,13 @@ NULL
 #'   split the input data for parallel processing.
 #'   This defaults to `p = x$dataStorage` and hence a per-file parallel
 #'   processing is applied for `Spectra` with file-based backends (such as the
-#'   [MsBackendMzR()]). The function returns a `Spectra` of length equal to the
-#'   unique levels of `f`.
+#'   [MsBackendMzR()]).
+#'   Prior combination of the spectra any processings queued in the lazy
+#'   evaluation queue are applied. Be aware that calling `combineSpectra` on a
+#'   `Spectra` object with an `MsBackendHdf5Peaks` backend will
```

Comment:
That is true for all backends with read and write support, isn't it? So maybe add a few words like: "Be aware ... with a backend that allows modifications, e.g. `MsBackendHdf5Peaks`, will **overwrite** ..."

Reply:
Thanks. I will add/update.